src/autorest-core/lib/approved-imports/jsonrpc.ts

```diff
@@ -0,0 +1,6 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export * from 'vscode-jsonrpc';
```

Comment:
their implementation seems to have some sort of cancel message https://github.com/Microsoft/vscode-languageserver-node/blob/master/jsonrpc/src/main.ts#L48 but I'm not sure if that's automatically built into every channel or something... will dig